docs/modules/deploy/pages/deploying-with-operator.adoc

```diff
@@ -137,6 +137,26 @@ You can configure the `expose-externally` field with the following properties:
 ** `NodePortNodeName`: Hazelcast clients are inside the same network as the Kubernetes nodes, so clients can access a member by the hostname of its Kubernetes node.
 ** `LoadBalancer`: Hazelcast clients are outside the Kubernetes network and can access members via the the external IP assigned to the LoadBalancer service.
 
+WARNING: Your Kubernetes cluster should be able to assign public IPs to LoadBalancer services to connect to Hazelcast from outside Kubernetes.
+
+WARNING: If NodePorts are used (`discoveryServiceType: NodePort` or `memberAccess: NodePortExternalIP`), then the Kubernetes nodes should have access from outside Kubernetes. This includes both the node IPs and ports.
+
+=== Things to consider in Google Kubernetes Engine (GKE):
+
+- The Kubernetes cluster should not be private.
+- If NodePorts are used (`discoveryServiceType: NodePort` or `memberAccess: NodePortExternalIP`), a firewall rule for each Hazelcast member should be added to allow TCP traffic on node ports.
+
+=== Things to consider in Amazon Elastic Kubernetes Service (EKS):
+
+- Having the VPC of the EKS cluster configured only with private subnets is not recommended.
+- If NodePorts are used (`discoveryServiceType: NodePort` or `memberAccess: NodePortExternalIP`), the VPC of the EKS cluster should be configured only with public subnets. Additionally, inbound rules for the node ports should be added to the nodes' security groups.
+- AWS load balancers are assigned with hostnames by default. Since it takes some time to resolve these hostnames, the member pods might fail to start with `CrashLoopBackOff` status in the beginning. But the pods will be running fine eventually after a couple of restarts.
+
+=== Things to consider in Azure Kubernetes Service (AKS):
+
+- The Kubernetes cluster should not be private.
+- If NodePorts are used (`discoveryServiceType: NodePort` or `memberAccess: NodePortExternalIP`), the node pools should be created by using `--enable-node-public-ip` flag. Additionally, the required inbound rules should be added for network security groups (NSGs) to allow access to node IPs and ports.
```

Comment:
Should we mention that it's not recommended as you've described [here](https://hazelcast.atlassian.net/wiki/spaces/CN/pages/3832283180/Expose+Externally+Feature+Testing+of+Hazelcast+Platform+Operator+for+EKS+GKE+and+AKS?focusedCommentId=3852140552#comment-3852140552)?

Reply:
I am not sure if we should mention it. The circumstances on AKS may change in future, so it might be better to leave the decision to the user. WDYT?

Reply:
Agree, it makes sense